packages/@headlessui-react/src/components/listbox/listbox-machine.ts

```diff
@@ -394,8 +397,9 @@ let reducers: {
 }
 
 export class ListboxMachine<T> extends Machine<State<T>, Actions<T>> {
-  static new({ __demoMode = false } = {}) {
+  static new({ __demoMode = false, id = globalThis.crypto.randomUUID() as string } = {}) {
```

Comment:
I was building this gradually and eventually decided to pass in an `id` so we can check the "same" id on the stack. But that also means that technically we are not even using the randomUUID here.

Thoughts on just removing it?